#goal definition
string movement_type
bool preempt
---
#result definition
bool success

---
#feedback
float64 completion